### PR TITLE
Enhancement/fm/create teamwork task

### PR DESCRIPTION
**Pull request by FahimehMirhaj**
Automating Task creation from iLab to Teamwork.

### CODE REVIEW

File: src/main/java/org/mskcc/limsrest/service/CustomForm.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+import java.util.HashMap;
+import java.util.Map;
+
+public class CustomForm {
+    private String id;
+    private String name;
+    private String note;
+    private Map<String, String> fields;
+
+    public CustomForm() {
```
Comment (dmcmanam) [inline src/main/java/org/mskcc/limsrest/service/CustomForm.java:12]: Add class comment to describe how it is used

File: src/main/java/org/mskcc/limsrest/service/PromoteBanked.java [hunk 2]
```diff
@@ -503,4 +524,170 @@ public String unaliquotName(String sampleName) {
...
+
+            String subject = requestId + " (" + numOfSamples + ")";
+            if(iLabComment != null &&  iLabComment.trim().length() > 0) {
+                subject += " #[IGO: iLab Comment]";
+            }
+            if (iLabComment != null)
+                message.setText(iLabComment + " \n#end");
+            message.setSubject(subject);
+
+            Transport.send(message);
```
Comment (dmcmanam) [inline src/main/java/org/mskcc/limsrest/service/PromoteBanked.java:623]: May need to check the return value, I'm not sure sending mail works on igolims in this way?

Comment (FahimehMirhaj) [reply src/main/java/org/mskcc/limsrest/service/PromoteBanked.java]: I am now using the workaround we use for pulling iLab into LIMS messaging. 
Now, at promote a text file is being written into  "/pskis34/vialelab/LIMS/AutomatedEmails/teamworkCard/" directory and a cron-job on igo-lims04 is picking text files to send emails to myself and IGO VMB Teamwork board.

File: src/main/java/org/mskcc/limsrest/service/CustomForm.java (hunk 1 above)
Comment (FahimehMirhaj) [reply src/main/java/org/mskcc/limsrest/service/CustomForm.java:17]: Addressed.

**Pull request merged**